packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -475,6 +503,20 @@ describe('expression', () => {
         expect(root.children[0].attrs.title.value).toBe('{myValue}');
     });
 
+    it('escaped brace characters', () => {
```

Comment:
```suggestion
    it('should correctly parse strings containing escaped curly brackets', () => {
```

Reply:
It's not escaped it's HTML encoded.

```suggestion
    it('should correctly parse strings containing HTML encoded curly brackets', () => {
```